index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+      this.items = [] ;
+      this.length = this.items.length ;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length; no
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < this.items.length && pos >= 0) {
+      return this.items[pos];
+    } else {
+      throw new Error('OutOfBounds');
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length > 0) {
+      return Math.max(...this.items);
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length > 0) {
+      return Math.min(...this.items);
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  sum() {}
+  sum() {
+    if (this.items.length > 0) {
+      return this.items.reduce((a, b) => a + b);
+    } else {
+      return 0;
+    }
+  }
 
-  avg() {}
+  avg() {
+    if (this.items.length > 0) {
+      return this.items.reduce((a, b) => a + b) / this.items.length;
```

Comment:
You could reuse your sum() here, because you're basically writing it again here.